node-yun-http/platformio.ini

```diff
@@ -0,0 +1,27 @@
+; PlatformIO Project Configuration File for Mois sensor node
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/en/latest/projectconf/
+
+[platformio]
+src_dir = .
+
+monitor_speed = 115200
+
+[common]
+lib_deps =
+    ../libraries/ADS1231
+    adafruit/Adafruit TSL2591 Library@1.4.3
+    adafruit/DHT sensor library@1.4.4
+    arduino-libraries/Bridge@1.7.0
+    milesburton/DallasTemperature@3.11.0
+    paulstoffregen/OneWire@2.3.5
+    robtillaart/RunningMedian@0.3.7
+    watterott/digitalWriteFast@1.0.0
```

Comment:
I deliberately used recent versions of the dependency libraries, and I am confident it will work. However, it would have to be tested to be absolutely sure.